pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/impl/TransactionBufferHandlerImpl.java

```diff
@@ -149,7 +149,6 @@ public void handleEndTxnOnTopicResponse(long requestId, CommandEndTxnOnPartition
             if (log.isDebugEnabled()) {
                 log.debug("[{}] Got end txn on topic response for for request {}", op.topic, response.getRequestId());
             }
-            log.info("[{}] Got end txn on topic response for for request {}", op.topic, response.getRequestId());
```

Comment:
this line is printed at debug level two lines above, probably is is a left over during development.

actually this line fills of spam broker logs during load testing